packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -148,3 +183,177 @@ class MemoryController {
     return false;
   }
 }
+
+class Settings {
+  String pattern = '*';
```

Comment:
maybe make 
```dart
pattern = '' 
```
indicate matching any to simplify this.

Reply:
Good idea.